### PR TITLE
CON-695: Change all kb links to new website

### PR DESCRIPTION
**Pull request by scvenema**

### CODE REVIEW

File: services/config.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@
 coreo_aws_rule "ec2-inventory-security-groups" do
   action :define
   service :ec2
-  link "http://kb.cloudcoreo.com/mydoc_all-inventory.html"
+  link "https://kb.securestate.vmware.com/aws-all-inventory.html"
```
[the remark this answers is not in the document]
Comment (armaneous) [reply services/config.rb]: It's a generic inventory link. Honestly, any rule with the name "inventory" in it isn't an actual rule and will be excluded from the generated JSON files.

File: services/config.rb [hunk 2]
```diff
@@ -252,7 +252,7 @@
 coreo_aws_rule "ec2-TCP-1521-0.0.0.0/0" do
   action :define
   service :ec2
-  link "http://kb.cloudcoreo.com/mydoc_ec2-tcpportopen.html"
+  link "https://kb.securestate.vmware.com/aws-ec2-tcpportopen.html"
```
[the remark this answers is not in the document]
Comment (ajayshekar) [reply services/config.rb]: +1

File: services/config.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (scvenema) [reply services/config.rb]: Agreed. The import into the new rules repo only occurs for rules which actually have QUERY strings so these 'inventory' type rules won't make the cut. Its my quick hack at replacement scripts that did this replacement irrespective of whether the rule is actually relevant anymore.

File: services/config.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (scvenema) [reply services/config.rb]: Looks like there is a generic kb article used for all of these TCP port-open type rules. Its currently in the `homeless-kb-docs` directory with the name `mydoc_ec2-tcpportopen.md`. The following rules could use this content:
```
   ec2-TCP-1433-0.0.0.0-0
   ec2-TCP-1521-0.0.0.0-0
   ec2-TCP-20
   ec2-TCP-21
   ec2-TCP-22-0.0.0.0-0
   ec2-TCP-23
   ec2-TCP-27017-0.0.0.0-0
   ec2-TCP-3306-0.0.0.0-0
   ec2-TCP-3389-0.0.0.0-0
   ec2-TCP-5432-0.0.0.0-0
   ec2-TCP-5439-0.0.0.0-0
   ec2-TCP-8080
```
An interesting side-effect of the new rules-content repo setup is that it expects different files for each rule. I've created #CON-807 to track the fix.

[the remark this answers is not in the document]
Comment (ajayshekar) [reply services/config.rb]: That is interesting! Maybe we can symlink the kb files that are shared across multiple rules?

[the remark this answers is not in the document]
Comment (scvenema) [reply services/config.rb]: Should be able to symlink. I just have to make sure that the docker container that is running the Jekyll compiler has access to the targets of the links in the correct context. Tracking that on CON-807.

**Pull request merged**